src/test/core/Coroutine_test.cpp

```diff
@@ -102,9 +106,13 @@ class Coroutine_test : public beast::unit_test::suite
     {
         using namespace std::chrono_literals;
         using namespace jtx;
-        Env env(*this);
+        Env env(*this, envconfig([](std::unique_ptr<Config> cfg) {
+            cfg->section("workers").legacy("1");
```

Comment:
Consider using the defined `SECTION_WORKERS` instead of "workers". Since we are only using a single JW thread, `Env env(*this);` is sufficient.